conjure-api/conjure/spec/unions.conjure.go

```diff
@@ -76,6 +76,20 @@ func (u *AuthType) UnmarshalYAML(unmarshal func(interface{}) error) error {
 	return safejson.Unmarshal(jsonBytes, *&u)
 }
 
+func (u *AuthType) AcceptFuncs(headerFunc func(HeaderAuthType) error, cookieFunc func(CookieAuthType) error, unknownFunc func(string) error) error {
+	switch u.typ {
+	default:
+		if u.typ == "" {
+			return fmt.Errorf("invalid value in union type")
```

Comment:
Is there a reason that that `""` is special-cased here rather than passing it to `unknownFunc`?

Reply:
I was following the precedent from `Accept` [below](https://github.com/palantir/conjure-go/pull/190/files#diff-6901295dd2f0f080dfaeb8e7d1600d638438e7db8a8e5adebd9e8f5e5a5e9fd3R96)

I defer to your opinion here, but I think we should be consistent between `Accept` and `AcceptFuncs` either way

Reply:
Ah thanks for the reference -- yes agree that should be consistent and this is fine.